src/test/java/org/jabref/logic/formatter/bibtexfields/EscapeCurrencySymbolsFormatterTest.java

```diff
@@ -0,0 +1,31 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class EscapeCurrencySymbolsFormatterTest {
+
+    private EscapeCurrencySymbolsFormatter formatter;
+
+    @BeforeEach
+    void setUp() {
+        formatter = new EscapeCurrencySymbolsFormatter();
+    }
+
+    @Test
+    void formatReturnsSameTextIfNoCurrencySymbolsPresent() throws Exception {
+        assertEquals("Lorem ipsum", formatter.format("Lorem ipsum"));
+    }
+
+    @Test
+    void formatEscapesCurrencySymbolsIfPresent() throws Exception {
+        assertEquals("Lorem\\\\$ipsum", formatter.format("Lorem$ipsum"));
```

Comment:
I think, these are two `\` too much. I think, the dollar sign should be replaced by `\$` and not by `\\$`

Reply:
Agree, thanks for help!